### PR TITLE
Add system info page

### PR DESCRIPTION
**Pull request by d-rita**
Adds the system info page with build information, modules and facility code. 

To test:
- Go to `\about` page by clicking on system info link in the right hand dropdown menu
- Click `update facility code` button if there is no facility code.
- Choose the right parameters in form and choose your facility accordingly.

### CODE REVIEW

**Comment by Mwanje** [comment]
@d-rita fix these conflicts

File: packages/esm-ugandaemr-app/src/pages/system-info/system-info.types.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  UgandaemrSync: string;
+  'App UI Module': string;
+  'Metadata Deploy': string;
+  'Reporting Compatibility': string;
+  'Bed Management Module': string;
+  'O3 Forms': string;
+  'HTML Widgets': string;
+  'Address Hierarchy': string;
+  'Data Entry Statistics': string;
+  'Registration Core Module': string;
```
Comment (pirupius) [inline packages/esm-ugandaemr-app/src/pages/system-info/system-info.types.tsx:26]: are these how it's being defined in the api? typically the format is 

interface SystemInfoTitleModuleInformation {
  systemInfo: string;
  idGeneration: string;
...
}

File: packages/esm-ugandaemr-app/src/pages/system-info/system-info.resource.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  useEffect(() => {
+    const dataFetch = async () => {
+      try {
+        const data = await (
+          await fetch(`${url}${param}&_pretty=true`, {
+            method: 'GET',
+            // headers: {
+            //   Authorization: `Basic ${btoa(`${username}:${password}`)}`,
+            //   'Content-Type': 'application/json',
+            // },
```
Comment (pirupius) [inline packages/esm-ugandaemr-app/src/pages/system-info/system-info.resource.tsx:41]: let's also do some cleanup to get rid of commented code that won't be used

**Review by pirupius — CHANGES_REQUESTED** [review]

File: packages/esm-ugandaemr-app/src/pages/system-info/system-info.types.tsx (hunk 1 above)
Comment (d-rita) [reply packages/esm-ugandaemr-app/src/pages/system-info/system-info.types.tsx:26]: The endpoint returns the information in that pattern so it is how i defined it.

File: packages/esm-ugandaemr-app/src/pages/system-info/system-info.resource.tsx (hunk 2 above)
Comment (d-rita) [reply packages/esm-ugandaemr-app/src/pages/system-info/system-info.resource.tsx]: Done the cleanup

**Review by pirupius — APPROVED** [review]

**Pull request merged**